src/EFCore.PG/Extensions/DbFunctionsExtensions/NpgsqlJsonDbFunctionsExtensions.cs

```diff
@@ -108,4 +108,22 @@ public static bool JsonExistAll(this DbFunctions _, object json, params string[]
     /// </remarks>
     public static string JsonTypeof(this DbFunctions _, object json)
         => throw new InvalidOperationException(CoreStrings.FunctionOnClient(nameof(JsonTypeof)));
+
+    /// <summary>
+    /// Returns true if the JSON value matches the JSON Path expression
```

Comment:
Better to just copy the PG docs:

```suggestion
    /// Checks whether the JSON path returns any item for the specified JSON value.
```